src/pages/docs/lexicon/user_card.md

```diff
@@ -0,0 +1,148 @@
+---
+title: User Card
+description: ""
+layout: "lexiconMigrationGuide"
+weight: 102
+
+clayComponentLink: "https://claycss.com/docs/components/cards.html"
+lexiconPattern: "User Card"
+lexiconPatternLink: "https://lexicondesign.io/docs/patterns/cards.html"
+---
+
+<article id="before-after">
+
+### How does it look like now (7.0)
+
+<img class="img img-thumbnail" src="/images/lexiconMigration/user_card_old.png">
+
+### Changes for Lexicon 2.0 (7.1)
+
+<img class="img img-thumbnail" src="/images/lexiconMigration/user_card_new.png">
+
+Change | Reason
+--- | ---
+① The **icons spritemap** has changed | Fit with the new Lexicon icons
+② The **link style** now is black, blue on hover. Also the entire card has hover state | Fit with new links styles and better visual feedback
```

Comment:
There are more changes like the initials (or the user image) appears now in the form of a sticker inside the card.